sway/sway-output.5.scd

```diff
@@ -36,7 +36,19 @@ must be separated by one space. For example:
 
 *output* <name> position|pos <X> <Y>
 	Places the specified output at the specific position in the global
-	coordinate space.
+	coordinate space. If scaling is active, it has to be considered when
+	positioning. For example, if the scaling factor for the left output is 2,
+	the relative position for the right output has to be divided by 2.
+
+	Example:
+
+		output HDMI1 scale 2
+
```

Comment:
Are double newlines are necessary in code blocks?

Reply:
Without the double newlines, it's just all on a single line when viewing the resulting man page